connectors/kafka_sink

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
```

Comment:
Not yet since it's "experimental" but as it graduates from that label, perhaps yes. There are some advantages to being directly linked into the wallaroo runtime though so I don't think we should make plans for that yet.